source/crud/tests/README.rst

```diff
@@ -57,6 +57,8 @@ Each YAML file has the following keys:
 
     - ``name``: The name of the operation as defined in the specification.
```

Comment:
I think you should mention that 'db-aggregate' means to run the database-level aggregate command.